app/modules/onboarding/features-usage.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
As I see it, this can be part of `elementor/app/modules/onboarding/module.php`, unless I am missing something?

Reply:
1. Could be, it's just for separation staff and better testing 
2. It was done in similar way in other places, see `app/modules/import-export/usage.php` `modules/notes/usage.php` etc

Reply:
Ok. If you prefer 👌  It just did not fit well with the onboarding module's current structure :)